testing/coreidle/test_main_c.cpp

```diff
@@ -400,6 +409,49 @@ TEST_P(coreidle_main_c_p, parse_err2) {
 }
 
 
+/**
+ * @test       lead_null_char
+ * @brief      Test: coreidle_main
+ * @details    When called with parameters that contains nullbytes<br>
+ *             it fails to find file and coreidle_main flags bitstream
+ *             invalid error. It returns 0 on cleanup success.<br>
+ */
+TEST_P(coreidle_main_c_p, lead_null_char) {
+  copy_bitstream("copy_bitstream.gbs");
+  char zero[32];
+  char one[32];
+  char two[32];
+  char *argv[] = { zero, one, two };
+
+  strcpy(zero, "coreidle");
+  strcpy(one, "-G");
+  strcpy(two, "\0 copy_bitstream.gbs");
+  EXPECT_EQ(coreidle_main(3, argv), 0);
+
+  unlink(copy_gbs_.c_str());
+}
+
+/**
+ * @test       mid_null_char
+ * @brief      Test: coreidle_main
+ * @details    When called with parameters that contains nullbytes<br>
+ *             it fails to find file and coreidle_main flags bitstream
+ *             invalid error. It returns 0 on cleanup success.<br>
+ */
+TEST_P(coreidle_main_c_p, mid_null_char) {
+  copy_bitstream("copy_bitstream.gbs");
+  char zero[32];
+  char one[32];
+  char two[32];
+  char *argv[] = { zero, one, two };
+
+  memset(two, 0, 32);
+  strcpy(two, "copy_bit\0stream.gbs");
+  EXPECT_EQ(coreidle_main(3, argv), 0);
```

Comment:
I wouldn't expect this to return 0, since the effective bitstream name "copy_bit" won't exist on the file system.